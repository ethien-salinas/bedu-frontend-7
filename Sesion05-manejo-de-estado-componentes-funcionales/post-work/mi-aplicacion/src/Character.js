import React from 'react'

import './Character.css'

function Character(props) {
  return (
    <div className="Character">
      <img src={props.image} alt={props.name} />
      <p>{props.name}</p>
      <p>{props.specie}</p>
    </div>
  )
}

export default Character
