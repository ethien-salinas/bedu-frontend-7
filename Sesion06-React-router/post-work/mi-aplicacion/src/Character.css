.Character {
  cursor: pointer;
  display: inline-block;
  margin: 1em;
  text-align: center;
}

.Character.Detail {
  cursor: default;
  display: grid;
  grid-template-columns: 300px 200px;
}
