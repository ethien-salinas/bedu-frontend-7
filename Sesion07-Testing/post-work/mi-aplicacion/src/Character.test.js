import React from 'react'

import Character from './Character'

test('Component renders without crash', () => {
  const component = shallow(<Character />)

  expect(component).toMatchSnapshot()
})
