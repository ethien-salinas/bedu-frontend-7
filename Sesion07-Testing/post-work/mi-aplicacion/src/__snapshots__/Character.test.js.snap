// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders without crash 1`] = `ShallowWrapper {}`;
